src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Locale;
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]).toLowerCase(Locale.ROOT);
```

Comment:
```suggestion
        return String.valueOf(Colors.values()[index]).toLowerCase();
```
I dont think we need it here, don't forget to remove import for Locale